examples/step-27/step-27.cc

```diff
@@ -379,7 +379,7 @@ namespace Step27
   void LaplaceProblem<dim>::solve()
   {
     SolverControl solver_control(system_rhs.size(),
-                                 1e-8 * system_rhs.l2_norm());
+                                 1e-16 * system_rhs.l2_norm());
```

Comment:
Is it guaranteed that this tolerance can actually be reached? I think this is substantially tighter than what we use elsewhere, right?

Reply:
I have not looked into this specific example, but of course this tolerance will not be reached by the true residual - nonetheless, the CG residual *estimate* as it gets updated during the iterations will happily go below 1e-16. But it is generally considered bad style to request less than say 1e-12 or 1e-13. Furthermore, the progress is a bit erratic beyond the level of roundoff, especially if the progress per iteration is modest (read: more than 50 iterations).

Reply:
You're right, of course. I only lowered it this far so that we got symmetric results.

We get very nearly (one entry is off by 1e-12) the same result with a factor of 1e-12 out front instead. I updated the code to use the more reasonable value.